transport/internal/tlsmux/listener_test.go

```diff
@@ -104,7 +152,15 @@ func TestMux(t *testing.T) {
 			lis, err := net.Listen("tcp", "127.0.0.1:0")
 			require.NoError(t, err, "unexpected error on listening")
 
-			muxLis := tlsmux.NewListener(lis, serverTlsConfig)
+			root := metrics.New()
+			muxLis := tlsmux.NewListener(tlsmux.Config{
+				Listener:      lis,
+				TLSConfig:     serverTlsConfig,
+				Meter:         root.Scope(),
+				Logger:        zap.NewNop(),
+				ServiceName:   "test-svc",
+				TransportName: "test-transport",
```

Comment:
Just trying to understand, so for every service & transport there will be a separate entry made in the tls config?

Reply:
I didn't get your question. Please let me know if below comment answers it: 
TLS config remains the same across transports, and every transport creates a mux listener if the tls config is available.